### ISSUE TITLE
Why do random effects intercepts from sjp.glmer differ from ranef?

### ISSUE THREAD

**Issue by CJustice77** [strengejacke/sjPlot#198]
Hi there. I'm trying to understand the output from the sjp.glmer function from the sjPlot package, and specifically, why the random effect intercept values differ from those produced by the ranef function from the lme4 package. I like the look of the random effects plot generated by sjp.glmer but I don't understand how to interpret the intercept values.  Here's some example code from the sjPlot web page:

# create binary response
sleepstudy$Reaction.dicho <-
dicho(sleepstudy$Reaction, dich.by = "median")

# fit model
fit <- glmer(Reaction.dicho ~ Days + (Days | Subject),
sleepstudy, family = binomial("logit"))

sjp.glmer(fit, sort.est = "(Intercept)", y.offset = .4)

# compare results from sjp.glmer with intercept values from ranef
ranef(fit)

It appears that the values from sjp.glmer are reported in log space, and therefore they are equal to exp(ranef(fit)$Subject[,1])

Why would the random effect intercepts be plotted in log space? If you're trying to show what the intercept values would be in real space instead of logit space, wouldn't you use the the inverse of the logit link function: exp(logit)/(1+exp(logit)), not exp(logit). Is this an error?

Thanks for your time and help.

Sincerely,

Casey Justice

**Comment by sjPlot** [comment]
`exp(logit)/(1+exp(logit))` is the same as `plogis(logit)`, and gives you the probability value (see `type = "pred"` or `type = "eff"`). As far as I understand, random effects can be interpreted a similar way as fixed effects, i.e. if you convert logits to odds ratio with `exp(logit-fixef)`, you could do the same for random effects, which gives you the odds ratio of a grouping level, with reference to the overall "odds".
